### DISCUSSION TITLE
Request for Voltage sensor Feature via Zigbee

### DISCUSSION THREAD

**Discussion by guillermohor** [pvvx/ZigbeeTLc#126] [Ideas]
Hello everyone,

I’ve been using devices powered by CR2032 lithium batteries, and I’ve noticed an issue when using the Zigbee version of the software. In the original firmware, it already shows the actual voltage of the battery, which is really helpful for accurately tracking the battery's life. However, in the Zigbee version, it only shows the battery percentage.

I think it would be great if the Zigbee version could include the voltage sensor as well, just like the original firmware. 

Thanks!

**Comment by pvvx** [comment]
![image](https://github.com/user-attachments/assets/d2481840-df8d-47f2-80fd-851c1076b155)

![image](https://github.com/user-attachments/assets/b7cea2d4-e1fc-4b13-81b7-437a2de72361)

ZigBee Cluster Library Specification
![image](https://github.com/user-attachments/assets/cb84b8f5-b9a6-4772-adbb-2d11ef7bc9b3)

**Comment by guillermohor** [comment]
Does it work in z2mqtt?
Thanks

**Comment by pvvx** [comment]
I don't use z2m.
For this issue, please contact https://www.zigbee2mqtt.io/.
Battery Information is a standard Zigbee option.
